docs/guides/ros2-gem.md

```diff
@@ -162,13 +163,25 @@ It is possible to implement your own control mechanisms with this Component.
 
 `ROS2SpawnerComponent` handles spawning entities during simulation.
 Available spawnables have to be set up as the component's field before the simulation.
+User is able to define named spawn points inside the Editor. This can be done by adding `ROS2SpawnPointComponent` to a child entity of an entity with `ROS2SpawnerComponent`.
 
-During the simulation user can access names of available spawnables and request spawning using ros2 services. The names of services are "/get_available_spawnable_names" and "/spawn_entity" respectivly. GetWorldProperties.srv and SpawnEntity.srv types from gazebo_msgs are used to handle these features.
+During the simulation user can access names of available spawnables and request spawning using ros2 services. 
+The names of services are `/get_available_spawnable_names` and `/spawn_entity` respectivly. 
+GetWorldProperties.srv and SpawnEntity.srv types are used to handle these features.
+In order to request defined spawn points names user can use `/get_spawn_points_names` service with GetWorldProperties.srv type.
+Detailed information about specific spawn point (e.g. pose) can be accessed using `/get_spawn_point_info` service with GetModelState.srv type.
+All used services types are defined in gazebo_msgs package.
```

Comment:
IMO, a better way to present this would be a table with rows: service name, service type, and description.